### PR TITLE
[Home] Optimise layout events and render cycles for generic grid

### PR DESCRIPTION
**Pull request by mennenia**
Down from 3-4 renders per rotation to just 1 🎉 

#skip_new_tests

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  No CHANGELOG added.
  </td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/ArtworkGrids/__tests__/GenericGrid-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "#skip_new_tests".
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,7 @@
 
 * Upgrade to React Native 0.54.0 - alloy
 * Follow Force and reduce image quality to 80 - alloy
+* Optimise the generic artwork grid for rotation = maxim
```
Comment (alloy) [inline CHANGELOG.md]: Trying to spruce things up, ey?

**Comment by alloy** [comment]
Awesome! Go ahead and merge after fixing that one `=` in the CHANGELOG.

File: CHANGELOG.md (hunk 1 above)
Comment (mennenia) [reply CHANGELOG.md]: I'm a rebel just for kicks yeah 🎶 

😄

**Pull request merged**